src/pages/Transactions/TransactionsTable.tsx

```diff
@@ -1,123 +1,213 @@
 import * as React from "react";
-import {useTheme} from "@mui/material";
+import {Box, Stack, useTheme} from "@mui/material";
 import * as RRD from "react-router-dom";
 import Link from "@mui/material/Link";
 import Table from "@mui/material/Table";
-import TableBody from "@mui/material/TableBody";
-import TableCell from "@mui/material/TableCell";
 import TableHead from "@mui/material/TableHead";
 import TableRow from "@mui/material/TableRow";
 import Typography from "@mui/material/Typography";
-import {useNavigate} from "react-router-dom";
+import {useNavigate, useParams} from "react-router-dom";
 import GeneralTableRow from "../../components/Table/GeneralTableRow";
 import GeneralTableHeaderCell from "../../components/Table/GeneralTableHeaderCell";
 import HashButton, {HashType} from "../../components/HashButton";
-
 import {Types} from "aptos";
 import {assertNever} from "../../utils";
 import {TableTransactionType} from "../../components/TransactionType";
 import {TableTransactionStatus} from "../../components/TransactionStatus";
-import {getFormattedTimestamp} from "../utils";
+import {getTableFormattedTimestamp} from "../utils";
 import GasFeeValue from "../../components/IndividualPageContent/ContentValue/GasFeeValue";
 import {useGetTransaction} from "../../api/hooks/useGetTransaction";
+import TransactionTypeTooltip from "../../components/Table/TransactionTypeTooltip";
+import {APTCurrencyValue} from "../../components/IndividualPageContent/ContentValue/CurrencyValue";
+import GeneralTableCell from "../../components/Table/GeneralTableCell";
+import GeneralTableBody from "../../components/Table/GeneralTableBody";
+import {
+  grey,
+  negativeColor,
+  primary,
+} from "../../themes/colors/aptosColorPalette";
+import TransactionFunction from "../Transaction/Tabs/Components/TransactionFunction";
+import {
+  getCoinBalanceChangeForAccount,
+  getTransactionAmount,
+  getTransactionCounterparty,
+} from "../Transaction/utils";
 
 type TransactionCellProps = {
   transaction: Types.Transaction;
 };
 
 function SequenceNumberCell({transaction}: TransactionCellProps) {
   return (
-    <TableCell sx={{textAlign: "left"}}>
+    <GeneralTableCell sx={{textAlign: "left"}}>
       {"sequence_number" in transaction && transaction.sequence_number}
-    </TableCell>
+    </GeneralTableCell>
   );
 }
 
-function TransactionStatusCell({transaction}: TransactionCellProps) {
+function TransactionVersionStatusCell({transaction}: TransactionCellProps) {
   return (
-    <TableCell sx={{textAlign: "left"}}>
-      {"success" in transaction && (
-        <TableTransactionStatus success={transaction.success} />
-      )}
-    </TableCell>
+    <GeneralTableCell sx={{textAlign: "left"}}>
+      <Stack direction="row" spacing={0.5}>
+        <Link
+          component={RRD.Link}
+          to={`/txn/${"version" in transaction && transaction.version}`}
```

Comment:
what happens if there is no version?

Reply:
theoretically it's not possible. every transaction other than pending transaction has a version. but our blockchain is fast so we almost never see a pending transaction (i've never seen it, i doubt that it exists)